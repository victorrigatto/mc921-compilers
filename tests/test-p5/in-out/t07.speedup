[SPEEDUP] Default: 121 Optimized: 103 Speedup: 1.17
