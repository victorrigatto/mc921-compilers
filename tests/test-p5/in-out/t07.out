370 is an Armstrong number.